RadixWallet/Clients/ProfileStore/ProfileStore+AsyncSequence+Updates.swift

```diff
@@ -0,0 +1,66 @@
+
+
+// MARK: Public
+extension ProfileStore {
+	/// The current network if any
+	public func network() async throws -> Profile.Network {
+		try profile.network(id: profile.networkID)
+	}
+
+	public var network: Profile.Network? {
+		profile.network
+	}
+
+	/// A multicasting replaying async sequence of distinct Profile.
+	public func values() async -> AnyAsyncSequence<Profile> {
```

Comment:
Same here, why async?

Reply:
Yes these should be removed. 

But the `async` here makes no difference from outside of actor isolation, i.e. `AccountsClient` still MUST write `let asyncSeq = await profileStore.values()` to get the async sequence, since `values` is an actor isolated method, and must remain so since it access (an actor isolated) stored property - `profileSubject` (all stored properties of an actor must be actor isolated).